helix-core/src/test/java/org/apache/helix/manager/zk/client/TestHelixZkClient.java

```diff
@@ -37,7 +37,7 @@
   @Test
   public void testZkConnectionManager() {
     final String TEST_ROOT = "/testZkConnectionManager/IDEALSTATES";
```

Comment:
I think the TEST_ROOT already contains the test method name.

Reply:
reverted,